web/src/app/app.module.ts

```diff
@@ -26,7 +29,7 @@ import { SharedModule } from './shared/shared.module';
     SharedModule.forRoot()
     ],
   exports: [],
-  providers: [],
+  providers: [ApiClientService],
```

Comment:
apiclientservice should be in corresponding module i.e. in service.module.ts

Reply:
Done